libraries/Bluefruit52Lib/src/BLEDiscovery.cpp

```diff
@@ -129,6 +130,9 @@ uint8_t BLEDiscovery::discoverCharacteristic(uint16_t conn_handle, BLEClientChar
     // timeout or has no data (due to GATT Error)
     if ( bytecount <= 0 ) break;
 
+    if ( disc_chr->count > MAX_DISC_CHARS ) // if we truncated the response, adjust the count to match
```

Comment:
can you have this simple if in the same line. I prefer either having single line if or with an full bracket one.

```
// if we truncated the response, adjust the count to match
if ( disc_chr->count > MAX_DISC_CHARS ) disc_chr->count = MAX_DISC_CHARS;
```